### PR TITLE
[N/A] [WIP] Queue screen

### PR DESCRIPTION
**Pull request by pedrofelipe**
screen shot 2016-10-13 at 6 10 23 pm
screen shot 2016-10-13 at 6 10 30 pm
screen shot 2016-10-13 at 6 11 01 pm

### CODE REVIEW

File: app/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+import $ from 'jquery';
+import jQuery from 'jquery';
```
Comment (gabrielrinaldi) [inline app/scripts/main.js]: Do we need both?

Comment (pedrofelipe) [reply app/scripts/main.js]: no, we don't. removing...

File: app/styles/components/_card.scss [hunk 2]
```diff
@@ -21,4 +17,64 @@
...
+    min-height: auto;
+
+    &__expand {
+      text-decoration: none;
+
+      &:focus {
+        outline: 0;
+      }
+
+      .is-expanded & {
```
Comment (viniciusalmeida) [inline app/styles/components/_card.scss]: Are you using [BEM](http://getbem.com/), right? Why not a modifier?

File: bower.json [hunk 3]
```diff
@@ -1,14 +0,0 @@
-{
```
Comment (viniciusalmeida) [inline bower.json:1]: It was a good change, but you should put this in other PR 🙂

Comment (gabrielrinaldi) [reply bower.json:1]: I agree

**Review by gabrielrinaldi — APPROVED** [review]
LGTM

File: app/scripts/card.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+import $ from 'jquery';
+
+$('.js-card-queue-toggle').click(function (e) {
+
+  e.preventDefault();
+
+  $(this).closest('.js-card-queue').toggleClass('mdl-card--expanded');
```
Comment (viniciusalmeida) [inline app/scripts/card.js:9]: In the future you can use this: https://github.com/viniciusalmeida/jQuery.toggleModifier 😉

**Review by viniciusalmeida — APPROVED** [review]

File: app/scripts/card.js (hunk 4 above)
Comment (pedrofelipe) [reply app/scripts/card.js:9]: Thanks! I'll add a card :-)

**Pull request merged**